src/js/components/DataFilters/stories/Simple.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
The 'paid' field in this story is a good example of why we may want to provide an out of the box alternative to having a RangeSelector. The values for the paid field range from 1,234 - 3,456. A keyboard only user can only move the range up or down by 1 value at a time by pressing or holding the left/right arrow key. When we have a range that spans a large set of values keyboard interaction is cumbersome.

I don't think we need to handle this in the current PR, just something to think about for future enhancements

Reply:
I've refactored ranges a bit. On the keyboard interaction aspect, I've added a `step` to restrict it to 20 steps. So, it shouldn't have the cumbersome one-at-a-time keyboard interaction.

I also am contemplating enhancing RangeSelector to include the min/max value text labelling, not for the first MVP version though.

Reply:
Nice, I like this solution. The keyboard behavior feels much better.

> I also am contemplating enhancing RangeSelector to include the min/max value text labelling, not for the first MVP version though.

I agree this can be handled separetly

Reply:
I'm wondering if we should restrict the `step` to only apply to keyboard users. Otherwise when I am using a mouse the interaction feels too restricted. Also there are cases where I end up with decimal values, I'm thinking we should round these.
<img width="361" alt="Screen Shot 2022-12-14 at 4 16 42 PM" src="https://user-images.githubusercontent.com/54560994/207735872-bb53cc45-ce40-4b1f-94eb-7cbe02ffbe34.png">

Reply:
I'm inclined to let this one go for now and see how the range filter gets used.